aws/data_source_aws_ecs_container_definition.go

```diff
@@ -59,6 +59,11 @@ func dataSourceAwsEcsContainerDefinition() *schema.Resource {
 				Computed: true,
 				Elem:     schema.TypeString,
 			},
+			"port_mappings": &schema.Schema{
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeMap, Elem: schema.TypeString},
```

Comment:
Since we know the structure, it would be better to set the schema rather than allowing unknown strings.
This would make us update the acc test also, to check given attributes :)